api/institutions/serializers.py

```diff
@@ -9,9 +9,10 @@ class InstitutionSerializer(JSONAPISerializer):
         'name'
     ])
 
-    name = ser.CharField(required=False)
-    id = ser.CharField(required=False, source='_id')
+    name = ser.CharField(required=True)
+    id = ser.CharField(required=True, source='_id')
     logo_path = ser.CharField()
+    auth_url = ser.CharField(required=True)
```

Comment:
`required=True` doesn't make sense here. `auth_url` is a static value; users should not be able to modify this. You should pass `read_only=True` to the appropriate fields.